### PR TITLE
Add "version" and "is_jenkins_jobs" params to constructor options

### PR DESCRIPTION
**Pull request by arcadefire**

### CODE REVIEW

File: src/main/groovy/net/rdrei/android/buildtimetracker/reporters/NetworkReporter.groovy [hunk 1]
```diff
@@ -7,13 +7,18 @@ class NetworkReporter extends AbstractBuildTimeTrackerReporter {
...
+
     NetworkReporter(
             Map<String, String> options,
             Logger logger,
             HttpClient httpClient
     ) {
         super(options, logger)
         this.httpClient = httpClient
+        this.isJenkinsJob = getOption("is_jenkins_job", false)
+        this.version = getOption("version", 1)
```
Comment (MyDogTom) [inline src/main/groovy/net/rdrei/android/buildtimetracker/reporters/NetworkReporter.groovy]: I'm not sure if it makes sense to specify version as an option. I'd say it should be hardcoded and correspond to plugin version.

**Review by MyDogTom — APPROVED** [review]
👍 I have only one concern.

File: src/main/groovy/net/rdrei/android/buildtimetracker/reporters/NetworkReporter.groovy (hunk 1 above)
Comment (arcadefire) [reply src/main/groovy/net/rdrei/android/buildtimetracker/reporters/NetworkReporter.groovy]: Good point!

**Review by MyDogTom — APPROVED** [review]
👍

**Pull request merged**